changelog/issue-3718.md

```diff
@@ -0,0 +1,5 @@
+audience: deployers
+level: major
```

Comment:
I think this would be `silent` per [this doc](https://github.com/taskcluster/taskcluster/blob/main/dev-docs/best-practices/changelog.md#levels).  It would be `minor` for "New feature users want to know about and take advantage of" except that the object service is disabled in deployments so users can't take advantage of it (yet).

Reply:
Fixed, thanks!